### PR TITLE
update to support launching windows store Subnautica 

### PR DESCRIPTION
**Pull request by silvervirus**

### CODE REVIEW

File: NitroxLauncher/LauncherLogic.cs [hunk 1]
```diff
@@ -298,7 +298,10 @@ private Process StartSubnautica()
             {
                 startInfo.FileName = "steam://run/264710";
             }
-
+          else if (PlatformDetection.IsMstore(subnauticaPath))
```
Comment (Sunrunner37) [inline NitroxLauncher/LauncherLogic.cs:301]: Let's fix the indentation in these files.  You may be mixing tabs and spaces.

File: NitroxLauncher/Patching/PlatformDetection.cs [hunk 2]
```diff
@@ -21,6 +21,14 @@ public static bool IsSteam(string subnauticaPath)
                 return true;
             }
             return false;
+        }
+		public static bool IsMstore(string subnauticaPath)
```
Comment (Sunrunner37) [inline NitroxLauncher/Patching/PlatformDetection.cs:25]: Let's be more verbose: `IsMicrosoftStore(...)`

Comment (silvervirus) [reply NitroxLauncher/Patching/PlatformDetection.cs:25]: alright thought i shorten it but alright thats fine

**Comment by silvervirus** [comment]
i did what i could

**Pull request merged**